portfolio/src/main/webapp/script.js

```diff
@@ -29,14 +29,28 @@ function addRandomFact() {
 
 
 function getFromServlet(){
-    fetch('/data').then(response => response.text()).then((saying) => { 
+    fetch('/data').then(response => response.text()).then((comment) => { 
 
-    const sayingContainer = document.getElementById('saying-container');
-    console.assert(sayingContainer, "You have no element with ID 'saying-container'"); 
-    sayingContainer.innerHTML = saying;
+    const commentContainer = document.getElementById('comment-container');
+    console.assert(commentContainer, "You have no element with ID 'comment-container'"); 
     
-    })
     
+    const list = document.getElementById('comment-container');
+    comment = comment.replace("[", "");
+    comment = comment.replace("]","");
+    commArr = comment.split(',');
+    for(i =0;  i < commArr.length; i++){
+        list.appendChild(createListElement(commArr[i]));
+    }
```

Comment:
This shouldn't be necessary.  If the response is of type "application/json", the comment variable will just be an array

Reply:
The content-type is "application/json" but my comments seemed to be one long string. When I used comment in the for-loop before conversion, it put each letter on a bulleted line.

Reply:
This will happen when you try to convert an array to a string.  If you just loop over the elements of the array instead you won't need to worry about this

Reply:
I tried to loop over the comment but got the fore-mentioned result, tried .forEach on comment and it said it wasn't a function, and then I took everything out and just did Array.isArray(comment) and it returned false. If I am receiving an array from /data, I'm not sure where I'm converting it.